src/components/createAppointment/RightCard.tsx

```diff
@@ -0,0 +1,93 @@
+import Image from "next/image";
+import BorderWithShadow from "../public/BorderWithShadow";
+import AddParticipant from "./AddParticipant";
+import { CardContent, Grid, Stack, Typography, Box, Button } from "@mui/material";
+import { User } from "@/types/User";
+
+type props = {
+  images: string[];
+  availableParticipants: User[];
+  selectedParticipants: User[];
+  onSubmit: () => void;
```

Comment:
`onSubmit` still void.

Reply:
IDK but 
[postId].tsx
(outter page which have this element (rightCard.tsx) need it)

Reply:
That line just defines the type of `onSubmit`, it's a function that take no argument and return nothing